doc/releases/nautilus.rst

```diff
@@ -84,6 +84,28 @@ Major Changes from Mimic
 
 - *CephFS*:
 
+  * MDS stability has been greatly improved for large caches and
+    long-running clients with a lot of RAM. Cache trimming and client
+    capability recall is now throttled to prevent overloading the MDS.
+  * CephFS may now be exported via NFS-Ganesha clusters in environments managed
+    by Rook. Ceph manages the clusters and ensures high-availability and
+    scalability.
```

Comment:
Link to blog post here? Or would that bitrot?

Reply:
it is appropriate, added!